BraveRewardsUI/Wallet/Notifications/RewardsNotificationViewBuilder.swift

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveRewards
+
+struct RewardsNotificationViewBuilder {
```

Comment:
This is much better! 

Would it be possible to get some unit tests for the `RewardsNotification` → `WalletActionNotification`/`WalletAlertNotification`/`WalletMessageNotification` logic?